features/step_definitions/local-time-difference-steps.js

```diff
@@ -1,8 +1,11 @@
   import { Given, Then } from 'cucumber';
 
   Given(/^I set wrong local time difference$/, async function () {
-    await this.client.execute(timeDifference => {
-      daedalus.api.ada.setLocalTimeDifference(timeDifference);
+    await this.client.executeAsync(timeDifference => {
+      daedalus.api.ada.setLocalTimeDifference(timeDifference)
+        .then(() => daedalus.stores.networkStatus._updateLocalTimeDifference())
+        .then(done)
```

Comment:
Btw. i just figured that this `done` will never be called because it's never defined 😆 i will fix that in the fork PR

Reply:
It is already fixed in: https://github.com/input-output-hk/daedalus/pull/726/files

Reply:
Btw. have you tried just returning the whole thing like this:
```
await this.client.executeAsync(timeDifference => {
    return daedalus.api.ada.setLocalTimeDifference(timeDifference)
      .then(() => daedalus.stores.networkStatus._updateLocalTimeDifference());
  }, 1511823600000);
```

Reply:
Nope :(
Feel free to improve it! Thanks :)